src/extension/serializer.ts

```diff
@@ -473,36 +473,23 @@ export class WasmSerializer extends SerializerBase {
   }
 }
 
-export class GrpcSerializer extends SerializerBase {
-  private client?: ParserServiceClient
-  protected ready: ReadyPromise
+// no common ancestor, any type used for protos
+export abstract class GrpcSerializerBase extends SerializerBase {
```

Comment:
As you suggested in the comment we should get rid of the GrpcSerializerBase class as we aren't using it anymore. 

You should try to make this as easy as possible for the runme folks to review but try to minimize the number of changes so they can focus their attention on the most substantitive changes.